src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
   ([#1427](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1427))
 * Propagators.DefaultTextMapPropagator will be used as the default Propagator
   ([#1427](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1428))
+* Removed Propagator from Options
```

Comment:
Removed Propagator from Instrumentation Options. Instrumentation now always respect the Propagator.DefaultTextMapPropagator.

Reply:
just updated!